packages/tasks/src/library-ui-elements.ts

```diff
@@ -422,7 +422,7 @@ const transformers = (model: ModelData) => {
 	if (!info) {
 		return [`# ⚠️ Type of model unknown`];
 	}
-	const remote_code_snippet = info.custom_class ? ", trust_remote_code=True" : "";
+	const remote_code_snippet = model.config?.auto_map ? ", trust_remote_code=True" : "";
```

Comment:
```suggestion
	const remote_code_snippet = model.tags.includes(TAG_CUSTOM_CODE) ? ", trust_remote_code=True" : "";
```

or just this then? might be more expressive?

Reply:
Ah yes, that's cleaner and delegates all to the Hub